src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -0,0 +1,67 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class WinningLotto {
+    private static final String DELIMITER = ",";
+
+    private final List<LottoNumber> lottoNumbers;
```

Comment:
```suggestion
    private final Lotto winningLottos;
```
`List<LottoNumber>`는 `Lotto`로 관리해볼 수 있겠네요 😄